### ISSUE TITLE
For Season 6

### ISSUE THREAD

**Issue by lolnotesftw** [bladecoding/LoLNotes#66]
Since Season 6 started and the new que, it does not give me any information before accepting the lobby, during the lobby when picking bans and champions. Only gives me information after the game starts when I have no chance to see what I am dealing with. Can you fix this please? Thank you very much in advance.

**Comment by mrxkenny** [comment]
Looks like the projects dead. It's been over a year since anyone has updated it.

**Comment by bladecoding** [comment]
The new queue system no longer provides team information before accepting the lobby. If there is still interest in getting teammate information during the lobby then I might update lolnotes to the new queue system.

**Comment by lolnotesftw** [comment]
Please do update lolnotes to the new queue system.
Date: Tue, 15 Mar 2016 10:25:48 -0700

**Comment by NitroXenon** [comment]
Please update this awesome tool.

**Comment by tr33s** [comment]
@NitroXenon [https://github.com/SebastienGllmt/LoLNotes](This) version has updated certificates. Doesn't support season 6 stuff though. 

Also was tracing chest stuff to try and add it to PVPNetConnect, but in LoLNotes it just says "operation: call " (blank)

**Comment by NitroXenon** [comment]
@tr33s Yeah I know. I don't use LoLNotes much now because FinalesFunkeln is awesome. Glad to know that the author of LoLNotes is alive :P

**Comment by bpeter289** [comment]
> The new queue system no longer provides team information before accepting the lobby. If there is still interest in getting teammate information during the lobby then I might update lolnotes to the new queue system.

I would definitely be interested in an update.
Also, is your PayPal still will@bladecoding.com? I would be more than happy to provide a donation.

Thanks as always.
